### DISCUSSION TITLE
Oriented Bounding Boxes

### DISCUSSION THREAD

**Discussion by TekayaNidham** [obss/sahi#464] [Ideas]
Hi,
First of all, thank you so much for the great work you've done so far!
Would it be possible to consider adapting the slicing approach to Oriented Bounding Boxes (x0y0,x1y1,x2y2,x3,y3).

**Comment by fcakyon** [comment]
Thanks for your interest @TekayaNidham !

Considering `sahi` has instance segmentation support, we can treat rotated bbox output as segmentation polygon and merge the sliced predictions as segmentation polygons. The output predictions may lose the 4 point rotated bbox structure but it would definitely work!

The most comprehensive rotated bbox library is `mmrotate`. Maybe we can look into mmrotate+sahi integration in the future 👍

**Comment by YoungjaeDev** [comment]
@fcakyon 
Is there any development idea and progress on that part?

**Reply by fcakyon** [reply]
Not atm. We currently accept related PRs.

**Reply by TekayaNidham** [reply]
@youngjae-avikus  I used this  [tool](https://github.com/ashnair1/dotadevkit), it has some limitations but it's good enough

**Reply by YoungjaeDev** [reply]
@TekayaNidham
Can you tell me which part you used and how you used it?
